code/modules/mob/floating_message.dm

```diff
@@ -19,8 +19,8 @@ var/global/list/floating_chat_colors = list()
 	var/fontsize = 6
 	if(small)
 		fontsize = 5
-	var/limit = 50
-	if(copytext(message, length(message) - 1) == "!!")
+	var/limit = 130
```

Comment:
130 feels a little too much for me, could end up cluttering the screen quite a bit. What about something in-between, in the 80-90ish area? How would that look?